test/unit/deployment.bats

```diff
@@ -1350,3 +1350,62 @@ load _helpers
   actual=$(echo "$object" | yq 'contains(["--kube-client-burst=2000"])' | tee /dev/stderr)
   [ "${actual}" = "true" ]
 }
+
+#--------------------------------------------------------------------
+# podDisruptionBudget
```

Comment:
Are there any tests we could add for the other fields you've added here as well? (priority class and topology spread constraints)